### PR TITLE
Recurrent_Neural_Networks

### PR DESCRIPTION
**Pull request by cserrano3**
It's necessary to fix the lsparse_softmax_cross_entropy_with_logits rank related to label series

### CODE REVIEW

**Comment by Hyalen** [comment]
Well, you`ll need to explain to me what`s going on step by step, because I didn`t get it. Hehehehe
